src/main/java/seedu/address/model/Aliases.java

```diff
@@ -0,0 +1,91 @@
+package seedu.address.model;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.NoSuchElementException;
+import java.util.Set;
+
+import seedu.address.logic.commands.AddCommand;
+import seedu.address.logic.commands.DeleteCommand;
+import seedu.address.logic.commands.EditCommand;
+import seedu.address.logic.commands.FindCommand;
+import seedu.address.logic.commands.HelpCommand;
+import seedu.address.logic.commands.ListCommand;
+import seedu.address.logic.commands.RedoCommand;
+import seedu.address.logic.commands.SelectCommand;
+import seedu.address.logic.commands.UndoCommand;
+
+/**
+ * Map of aliases to their commands
+ */
+public class Aliases {
+
+    private final Map<String, String> map = new HashMap<>();
+
+    /*
+     * We initialise the map with aliases for frequently used commands. Users can add other aliases themselves.
+     */
+    {
+        map.put("a", AddCommand.COMMAND_WORD);
+        map.put("d", DeleteCommand.COMMAND_WORD);
+        map.put("e", EditCommand.COMMAND_WORD);
+        map.put("f", FindCommand.COMMAND_WORD);
+        map.put("h", HelpCommand.COMMAND_WORD);
+        map.put("l", ListCommand.COMMAND_WORD);
+        map.put("r", RedoCommand.COMMAND_WORD);
+        map.put("s", SelectCommand.COMMAND_WORD);
+        map.put("u", UndoCommand.COMMAND_WORD);
+    }
```

Comment:
It seems like this is loaded on creation of Aliases object. User is unable to permanently remove these default aliases as its reloaded on app start.